File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbNews.java
package com.alvinhkh.buseta.nlb.model;

import com.google.gson.annotations.SerializedName;

public class NlbNews {

    @SerializedName("newsId")
    public String newsId;

    @SerializedName("title")
    public String title;

    @SerializedName("content")
    public String content;

    @SerializedName("publishDate")
    public String publishDate;

    public String toString() {
        return "NlbNews{newsId=" + newsId + ", title=" + title + ", content=" + content +
                ", publishDate=" + publishDate + "}";
    }

}

File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbNews.kt
package com.alvinhkh.buseta.nlb.model

import com.google.gson.annotations.SerializedName

data class NlbNews(
        @SerializedName("newsId")
        var newsId: String = "",

        @SerializedName("title")
        var title: String = "",

        @SerializedName("content")
        var content: String = "",

        @SerializedName("publishDate")
        var publishDate: String = ""
)